lib/zold/node/farm.rb

```diff
@@ -165,11 +165,18 @@ def save(threads, list = [])
     def load
       @mutex.synchronize do
         if File.exist?(@cache)
-          AtomicFile.new(@cache).read.split(/\n/).map { |t| Score.parse(t) }
+          AtomicFile.new(@cache).read.split(/\n/).map { |t| parse_score_line(t) }
         else
           []
         end
       end
     end
+
+    def parse_score_line(line)
+      Score.parse(line)
+    rescue StandardError => e
+      @log.error(e.message)
+      nil
```

Comment:
@emilianodellacasa `nil` is evil, no matter what: https://www.yegor256.com/2014/05/13/why-null-is-bad.html Stop even thinking about using it. Instead, I would suggest to make this method return an array of scores found in the line. Sometimes the array will have a single element, sometimes it will be empty. More about it: https://www.yegor256.com/2018/05/22/default-arguments-against-null.html